### PR TITLE
Update README test badge

### PR DESCRIPTION
**Pull request by danielrozenberg**

### CODE REVIEW

**Comment by nhant01** [comment]
@danielrozenberg 
Do you know if the previous merge created a new version of the jar? Otherwise, please update the pom.xml to 1.0.23 for this merge, from <version>1.0.22</version> to  <version>1.0.23</version>.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # AMP HTML ⚡ Validator
-Build status: [![build_status](https://travis-ci.com/ampproject/validator-java.svg?branch=master)](https://travis-ci.com/ampproject/validator-java)
+
+Build status: ![test](https://github.com/ampproject/validator-java/actions/workflows/test.yml/badge.svg?event=push)
```
Comment (nhant01) [inline README.md:3]: What would be the link to CI for diagnosing problems? We always logged into travis-ci.com/ampproject to investigate when we have a build failure.

Comment (danielrozenberg) [reply README.md]: Surrounded the badge with a link to GitHub Actions

**Comment by danielrozenberg** [comment]
> Do you know if the previous merge created a new version of the jar? Otherwise, please update the pom.xml to 1.0.23 for this merge, from 1.0.22 to 1.0.23.

The build failed due to credential issues, I messaged you on Slack so we can resolve this

**Comment by nhant01** [comment]
> > Do you know if the previous merge created a new version of the jar? Otherwise, please update the pom.xml to 1.0.23 for this merge, from 1.0.22 to 1.0.23.
> 
> The build failed due to credential issues, I messaged you on Slack so we can resolve this

**Comment by nhant01** [comment]
> > Do you know if the previous merge created a new version of the jar? Otherwise, please update the pom.xml to 1.0.23 for this merge, from 1.0.22 to 1.0.23.
> 
> The build failed due to credential issues, I messaged you on Slack so we can resolve this

@danielrozenberg I sent you in the slack.

**Pull request merged**